core/jazz_create-serverless-service/index.js

```diff
@@ -208,29 +208,32 @@ var getServiceData = (service_creation_data, authToken, configData) => {
             "DESCRIPTION": service_creation_data.description,
             "TYPE": service_creation_data.service_type,
             "RUNTIME": service_creation_data.runtime,
-            "REGION": service_creation_data.region,
             "USERNAME": user_id,
             "IS_PUBLIC_ENDPOINT": service_creation_data.is_public_endpoint || false,
             "STATUS": "creation_started"
         };
 
         var serviceMetadataObj = {};
         if (service_creation_data.tags) {
-            inputs.TAGS = service_creation_data.tags;
+          inputs.TAGS = service_creation_data.tags;
         }
 
         if (service_creation_data.email) {
-            inputs.EMAIL = service_creation_data.email;
+          inputs.EMAIL = service_creation_data.email;
         }
 
         if (service_creation_data.slack_channel) {
-            inputs.SLACKCHANNEL = service_creation_data.slack_channel;
+          inputs.SLACKCHANNEL = service_creation_data.slack_channel;
         }
 
         if ((service_creation_data.service_type === "api" || service_creation_data.service_type === "function") && (service_creation_data.require_internal_access)) {
-            serviceMetadataObj.require_internal_access = service_creation_data.require_internal_access;
+          serviceMetadataObj.require_internal_access = service_creation_data.require_internal_access;
         }
 
+        serviceMetadataObj.accountId = service_creation_data.accountId || configData.ACCOUNTID
```

Comment:
@SiniWilson Since we'll support multiple account  & regions, this structure might not work.  @rjsughosh  & I discussed this briefly. We'll go with the following format - 

```json
service_creation_data.deployment_accounts=[
  {
    "accountId": "12345679",
    "region": "us-west-2",
    "provider": "aws",
    "primary": true
  },
  {
    "accountId": "987654",
    "region": "us-east-1",
    "provider": "aws"
  }
]
```

Reply:
You will require changes to  swagger for  this service since we should accept the region/account/provider  values from API caller. Can you discuss with @rjsughosh  on the changes that we have done in the internal?

Reply:
done